source/inmemory.rst

```diff
@@ -1,111 +1,101 @@
-along with the standard MongoDB engines
-(the original `MMAPv1`_ and the default `WiredTiger`_),
-as well as `MongoRocks`_.
+The *Percona Memory Engine* is available in |PSMDB| along with the standard
+MongoDB engines (the original `MMAPv1`_ and the default `WiredTiger`_).
+
 
 Using Percona Memory Engine
-===========================
+================================================================================
 
-As of version 3.2, |PSMDB| runs with `WiredTiger`_ by default.
-You can select a storage engine
-using the ``--storageEngine`` command-line option when you start ``mongod``.
-Alternatively, you can set the ``storage.engine`` variable
-in the configuration file (by default, :file:`/etc/mongod.conf`):
+As of version 3.2, |PSMDB| runs with `WiredTiger`_ by default.  You can select a
+storage engine using the ``--storageEngine`` command-line option when you start
+``mongod``.  Alternatively, you can set the ``storage.engine`` variable in the
+configuration file (by default, :file:`/etc/mongod.conf`):
 
-Data created by one storage engine
-is not compatible with other storage engines,
-because each one has its own data model.
-When changing the storage engine, you have to do one of the following:
+Data created by one storage engine is not compatible with other storage engines,
+because each one has its own data model.  When changing the storage engine, you
+have to do one of the following:
 
-* If you simply want to temporarily test Percona Memory Engine,
-  change to a different data directory with the ``--dbpath``
-  command-line option:
+* If you simply want to temporarily test Percona Memory Engine, change
+  to a different data directory with the ``--dbpath`` command-line
+  option. Make sure that the user running |mongod| has read and write
+  permissons for the new data directory.
 
   .. code-block:: bash
 
      $ service mongod stop
      $ mongod --storageEngine inMemory --dbpath <newDataDir>
 
-  .. note:: Make sure that the user running ``mongod``
-     has read and write permissons for the new data directory.
+  .. note::
 
-  .. note:: Although *Percona Memory Engine* stores all data in memory,
+     Although *Percona Memory Engine* stores all data in memory,
      some diagnostics and statistics are still written to disk.
      This is controlled using
      the :option:`--inMemoryStatisticsLogDelaySecs` option.
 
-* If you want to permanently switch to Percona Memory Engine
-  and do not have any valuable data in your database,
-  clean out the default data directory and edit the configuration file:
+* If you want to permanently switch to Percona Memory Engine and do not have any
+  valuable data in your database, clean out the default data directory and edit
+  the configuration file:
 
   .. code-block:: bash
 
      $ service mongod stop
      $ rm -rf /var/lib/mongodb/*
-     $ sed -i '/engine: \*inMemory/s/#//g' /etc/mongod.conf
+     $ sed -i '/engine: *inMemory/s/#//g' mongod.conf
```

Comment:
As discussed on slack, this should work with or without quotes if you not decided to remove this altogether:
```
sed -i '/engine: .*inMemory/s/#//g' /etc/mongod.conf
```

Reply:
I'll leave the updated version of the sed command. Later, i'll check the documentation for such low level commands to make sure they are used consistently.